application-task-ui/src/main/resources/TaskManager/TaskManagerSheet.xml

```diff
@@ -334,6 +335,12 @@
         var status = $(this).val();
         if (status == "Done") {
           $("input[name='TaskManager.TaskManagerClass_0_progress']").val(100);
+        } else if (status == "InProgress") {
+          let startDateInput = $("input[name='TaskManager.TaskManagerClass_0_startDate']");
+          const format = moment().toMomentFormatString(startDateInput.data("format"));
+          if (startDateInput.val() == "") {
+            startDateInput.val(moment().format(format));
```

Comment:
Here you are simply the current date as the start date when status was changed to in progress?

Reply:
The ```startDate``` field was added in this PR, so could it be considered as part of the Gantt update?

Reply:
Yes, it's ok to be part of this PR